package.json

```diff
@@ -50,27 +50,37 @@
 	"scripts": {
 		"vscode:prepublish": "npm run compile",
 		"compile": "tsc -p ./",
-		"watch": "tsc -watch -p ./",
+		"watch": "concurrently \"rollup -c -w\" \"tsc -watch -p ./\"",
```

Comment:
Oh I see you added it to the watch command, cool.

Reply:
Haha...Yeah.